phoenix-core/src/main/java/org/apache/phoenix/mapreduce/index/IndexUpgradeTool.java

```diff
@@ -502,6 +501,21 @@ private int rebuildIndexes(String dataTable, IndexTool indexingTool) {
         return 0;
     }
 
+    private String[] getArgValues(String schema, String baseTable, String indexName,
```

Comment:
Also, perhaps rename to getIndexToolArgValues?

Reply:
I got this idea while I was going through one of the ITs, it was named such and hence picked for consistency. But, I get your point.